airflow/providers/amazon/aws/hooks/chime.py

```diff
@@ -0,0 +1,104 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""This module contains a web hook for Chime."""
+from __future__ import annotations
+
+import json
+import re
+from typing import Any
+
+from airflow.exceptions import AirflowException
+from airflow.providers.http.hooks.http import HttpHook
+
+
+class ChimeWebhookHook(HttpHook):
+    """Interact with Chime Web Hooks to create notifications.
+
+    .. warning:: This hook is only designed to work with Web Hooks and not Chatbots.
+
+    :param http_conn_id: Http connection ID with host as "https://hooks.chime.aws" and
+                         default webhook endpoint in the extra field in the form of
+                         {"webhook_endpoint": "incomingwebhooks/{webhook.id}?token{webhook.token}"}
+    :param webhook_endpoint: Chime webhook endpoint in the form of
+                             "incomingwebhooks/{webhook.id}?token={webhook.token}"
+    :param message: The message you want to send to your Chime room.
```

Comment:
This is not a parameter of `__init__` method. Please move it to `send_message` method